snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -76,6 +79,93 @@ def __init__(self, *, output, source, project_options,
             os.path.join('~', 'snap', 'lxd', 'common'))
         os.makedirs(self._lxd_common_dir, exist_ok=True)
 
+    def _get_multipass_remote(self):
+        try:
+            containers = subprocess.check_output([
+                'multipass', 'list']).decode()
+            # Trailing space to disambiguate names starting with snapcraft
+            if 'snapcraft ' not in containers:
+                subprocess.check_call([
+                    'multipass', 'launch', '--name=snapcraft', '--disk', '10G',
+                    ])
+            subprocess.check_call([
+                'multipass', 'start', 'snapcraft'])
+            self._setup_lxd()
+
+            return 'multipass'
+        except FileNotFoundError:
+            raise ContainerError('Multipass is not installed')
```

Comment:
According to the guidelines from mpt, we should say how to fix the error.
So maybe here we should say:
`Run sudo snap install multipass --beta --classic`. Or once we support all the other OSs, give a URL to the install instructions.